templates/js/custom-copilot-assistant-assistants-api/.vscode/launch.json.tpl

```diff
@@ -75,6 +75,30 @@
                 "order": 3
             },
             "internalConsoleOptions": "neverOpen",
+        {{#CEAEnabled}}
+        },
+        {
+            "name": "Launch in Microsoft 365 app (Edge)",
+            "type": "msedge",
+            "request": "launch",
+            "url": "https://www.office.com/chat",
+            "presentation": {
+                "group": "all",
+                "hidden": true
```

Comment:
need to remove `hidden` settings to enable remote launch option, also need to add `extend to M365` action in teamsapp.yml. Please refer to latest changes in pr https://github.com/OfficeDev/teams-toolkit/pull/12593